blueman/main/Manager.py

```diff
@@ -166,26 +169,26 @@ def on_dbus_name_appeared(_connection, name, owner):
 
         self.window.present_with_time(Gtk.get_current_event_time())
 
-    def _on_configure(self, window, event):
+    def _on_configure(self, _window: Gtk.Widget, event: Gdk.Event) -> bool:
```

Comment:
Should be ``Gtk.ApplicationWindow``  If we don't get one of them we have a big problem :-).